portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,45 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="navbar">
+        <div>
+            <h1 class="name">L I L Y</h1>
+            <h1 class="name">O U</h1>
+        </div>
+        <div id = "social">
+            <a href="https://www.linkedin.com/in/lily-o-2897b0121/"><img src="images/linkedin.png" alt="linkedin logo" width="30"></a>
```

Comment:
Another optional thing: Technically the Google HTML style guide recommends indenting using two spaces, and wrapping long lines: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

I personally don't have a strong opinion though, so I'm happy to leave it up to you whether you want to follow the style guide strictly or not!

Reply:
I'll make those changes. I like consistent spacing and good readability.